<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,1">
    <item>
     <widget class="QWidget" name="left_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTreeView" name="treeView">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="center_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>12</horstretch>
        <verstretch>12</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>3</width>
        <height>3</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="right_widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="treeView_2"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu123">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actiondaoru"/>
    <addaction name="actiondaochu"/>
    <addaction name="actionxinxi"/>
   </widget>
   <widget class="QMenu" name="menubianji">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionduibi"/>
    <addaction name="actiontuxiang"/>
    <addaction name="actiontuxiang_2"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu123"/>
   <addaction name="menubianji"/>
   <addaction name="menu"/>
  </widget>
  <action name="actiondaoru">
   <property name="text">
    <string>文件导入</string>
   </property>
  </action>
  <action name="actiondaochu">
   <property name="text">
    <string>文件导出</string>
   </property>
  </action>
  <action name="actionxinxi">
   <property name="text">
    <string>文件信息</string>
   </property>
  </action>
  <action name="actionduibi">
   <property name="text">
    <string>对比度拉伸</string>
   </property>
  </action>
  <action name="actiontuxiang">
   <property name="text">
    <string>图像旋转</string>
   </property>
  </action>
  <action name="actiontuxiang_2">
   <property name="text">
    <string>图像翻转</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>处理结果</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
